app/views/static/evacuate-form.html.haml

```diff
@@ -46,7 +46,6 @@
           %select{:name        => 'destination_host',
                   'ng-model'   => 'vm.vmCloudModel.host',
                   'ng-options' => 'host.name as host.name for host in vm.hosts track by host.id'}
-%div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
-                'paging_div_buttons_state_enabled' => true,
-                'paging_div_buttons_id'            => "angular_paging_div_buttons",
-                'paging_div_buttons_type'          => "Submit"}
+
+.clearfix
+  = render :partial => 'layouts/angular/custom_form_buttons_controller_as', :locals => {:button_label => _('Submit'), :button_click => 'vm.submitClicked()'}
```

Comment:
this puts the buttons inside the clearfix class
wrong indent, I think

Reply:
What can go wrong if we put buttons inside the clearfix class?

Reply:
¯\\_(ツ)_/¯

the worst thing that can happen is no buttons visible,
the more likely thing is `clearfix` won't do its job, definitely if anything inside floats.

Reply:
with "wrong" indent:
![with](https://user-images.githubusercontent.com/13417815/78588437-eb58eb00-783e-11ea-8a04-5661bc2c4c18.png)
without "wrong" indent:
![without](https://user-images.githubusercontent.com/13417815/78588468-fb70ca80-783e-11ea-874a-c0b0d5305af7.png)
I've tried many other things to achieve the first picture, only this weird thing with putting it to the clearfix class worked. Also buttons work there as expected and displaying the buttons when changing size of the screen was also ok.

Reply:
that looks like you want the clearfix *after* the buttons, not before

Reply:
Right, it works :+1: :confetti_ball: . I just considered trying this useless because `custom_form_buttons_controller_as` already contains `.clearfix` at the beginning of the file :smile: